src/modules/member/actionsType.js

```diff
@@ -0,0 +1,18 @@
+import keyMirror from 'fbjs/lib/keyMirror'
+
+export default keyMirror({
+    RECIVE_MEMBERLIST_START: null,
+    RECIVE_MEMBERLIST_SUCCESS: null,
+    RECIVE_MEMBERLIST_FAILURE: null,
+    RECIVE_MEMBER_START: null,
+    RECIVE_MEMBER_SUCCESS: null,
+    RECIVE_MEMBER_FAILURE: null,
+    RECIVE_MEMBERSEARCH_START: null,
+    RECIVE_MEMBERSEARCH_SUCCESS: null,
+    RECIVE_MEMBERSEARCH_FAILURE: null,
+  RECIVE_MEMBERSTATS_START: null,
```

Comment:
Krøll på indentering her. Er vel strengt tatt de andre som har for mye indentering. 2 spaces skal det være per indentering